io.itch.itch.yaml

```diff
@@ -1,6 +1,6 @@
 app-id: io.itch.itch
 runtime: org.freedesktop.Platform
-runtime-version: '22.08'
+runtime-version: '23.08'
 sdk: org.freedesktop.Sdk
 base: org.winehq.Wine
 base-version: stable-22.08
```

Comment:
@Doomsdayrs Why did you keep base (Wine) at 22.08?

Reply:
Because they did not update wine at that point

Reply:
Fixed with #12